### ISSUE TITLE
Tree view should keep unfold state

### ISSUE THREAD

**Issue by jamhsueh** [DotJoshJohnson/vscode-xml#360]
**Description**
So I have two xml documents and I was trying to compare the structure using the tree view. What terrible is, when I switched to the other file and switched back, I found that the tree view is folded and I have to unfold it again.
Is it possible to keep the unfold state of a tree or restore it when switch between files?

**Current Workarounds**
rfcosta's suggestion.

**Comment by rfcosta** [comment]
Try use different instances of VsCode so you switch instances not Windows. Just a guess...

**Comment by jamhsueh** [comment]
> Try use different instances of VsCode so you switch instances not Windows. Just a guess...

Yes, its a workaround. But still, I think the feature I mentioned is necessary and make sence.
